### PR TITLE
Add UnitTest for FormView

### PR DESCRIPTION
**Pull request by bartholdbos**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Notice there is a missing test for the FormView class

### CODE REVIEW

File: src/Symfony/Component/Form/Tests/FormViewTest.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        $this->formView->offsetSet('', '');
+    }
+
+    public function testGeneric()
+    {
+        self::assertFalse($this->formView->isMethodRendered());
+
+        $this->formView->setMethodRendered();
+
+        self::assertTrue($this->formView->isMethodRendered());
```
Comment (Tobion) [inline src/Symfony/Component/Form/Tests/FormViewTest.php]: we don't use static calls for phpunit assertions

File: src/Symfony/Component/Form/Tests/FormViewTest.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    private $formView;
+
+    protected function setUp()
+    {
+        $this->formView = new FormView();
+    }
+
+    public function testOffsetSet()
+    {
+        self::expectException(BadMethodCallException::class);
```
Comment (chalasr) [inline src/Symfony/Component/Form/Tests/FormViewTest.php]: we use the `@expectedException` annotation

**Comment by bartholdbos** [comment]
The failing AppVeyor build is probably not related to my changes

File: src/Symfony/Component/Form/Tests/FormViewTest.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+class FormViewTest extends TestCase
+{
+    /**
+     * @var FormView
+     */
+    private $formView;
+
+    protected function setUp()
+    {
+        $this->formView = new FormView();
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Tests/FormViewTest.php]: I don't think this is usefull: the instantiation should be done inline in each test case IMHO;

**Review by nicolas-grekas — COMMENTED** [review]
Shouldn't this target 2.7? Can you please rebase on 2.7 + retarget on github?

File: src/Symfony/Component/Form/Tests/FormViewTest.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+    /**
+     * @expectedException \BadMethodCallException
+     */
+    public function testOffsetSet()
+    {
+        $this->formView->offsetSet('', '');
+    }
+
+    public function testGeneric()
```
Comment (fabpot) [inline src/Symfony/Component/Form/Tests/FormViewTest.php]: Test names should be explicit about what we are testing.

**Comment by nicolas-grekas** [comment]
Status: needs work

**Comment by fabpot** [comment]
Closing as there is no more activity.

**Pull request closed without merging**